cmd/minikube/cmd/addons/enable.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package addons
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/docker/machine/libmachine"
+	"github.com/docker/machine/libmachine/drivers"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"k8s.io/minikube/cmd/minikube/cmd/config"
+	"k8s.io/minikube/pkg/minikube/assets"
+	"k8s.io/minikube/pkg/minikube/cluster"
+	"k8s.io/minikube/pkg/minikube/constants"
+	"k8s.io/minikube/pkg/minikube/sshutil"
+)
+
+var addonsEnableCmd = &cobra.Command{
+	Use:   "enable ADDON_NAME",
+	Short: "Enables the value of ADDON_NAME from the minikube addons file",
+	Long:  "Returns the value of ADDON_NAME from the minikube addons file.  Can be overwritten at runtime by flags or environmental variables.",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 1 {
+			fmt.Fprintln(os.Stderr, "usage: minikube addons enable ADDON_NAME")
+			os.Exit(1)
+		}
+
+		addon := args[0]
+		err := enable(addon)
+		if err != nil {
+			fmt.Fprintln(os.Stdout, err)
+		}
+		fmt.Fprintln(os.Stdout, fmt.Sprintf("%s was successfully enabled", addon))
+	},
+}
+
+func init() {
+	AddonsCmd.AddCommand(addonsEnableCmd)
+}
+
+func enable(name string) error {
+	if addon, ok := assets.Addons[name]; ok {
+		// check if already enabled? (not sure if better or worse....)
+		if err := config.Set(name, "true"); err != nil {
+			return err
+		}
+		//transfer addon to VM if VM is on...
+		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
+		defer api.Close()
+		cluster.EnsureMinikubeRunningOrExit(api, 0)
+
+		host, err := cluster.CheckIfApiExistsAndLoad(api)
```

Comment:
I know you just moved a bunch of stuff from pkg/ but --- 

Maybe `enable` function should be in our pkg/cluster so we don't have to expose CheckIfApiExistsAndLoad.  I don't think its a huge deal but we should probably err on the side of trying not to expose this if we don't have to.